src/Components/ExternalResult/ResultList.tsx

```diff
@@ -269,7 +248,7 @@ export default function ResultList() {
 
   let resultList: any[] = [];
   if (data && data.length) {
-    resultList = data.map((result: any) => {
+    resultList = data.map((result: any, _idx: number) => {
```

Comment:
```suggestion
    resultList = data.map((result: any) => {
```

if we are anyways not going to use index (idx), it would be better of not declaring it

Reply:
Removed the unused variable